lib/src/tasks/coverage/config.dart

```diff
@@ -17,6 +17,7 @@ library dart_dev.src.tasks.coverage.config;
 import 'package:dart_dev/src/tasks/config.dart';
 
 const bool defaultHtml = true;
+const List<String> defaultFunctionalTests = const [];
```

Comment:
Looks like you do actually have this in the test task configuration, so you should be able to use that one instead of this